tensorboard/webapp/runs/views/runs_table/runs_group_menu_button_component.ts

```diff
@@ -24,18 +32,58 @@ import {ChangeDetectionStrategy, Component} from '@angular/core';
     >
       <mat-icon svgIcon="palette_24px"></mat-icon>
     </button>
-    <mat-menu #groupByMenu="matMenu">
-      <button mat-menu-item>
-        <span>Experiments</span>
+    <mat-menu #groupByMenu="matMenu" class="run-table-color-group-by">
+      <div class="label">Color runs by</div>
+      <button
+        mat-menu-item
+        role="menuitemradio"
+        (click)="onGroupByChange.emit({key: GroupByKey.EXPERIMENT})"
+      >
+        <span>
+          <mat-icon
+            *ngIf="selectedGroupBy.key === GroupByKey.EXPERIMENT"
+            svgIcon="done_24px"
+          ></mat-icon>
+        </span>
+        <label>Experiments</label>
       </button>
-      <button mat-menu-item>
-        <span>Runs</span>
+      <button
+        mat-menu-item
+        role="menuitemradio"
+        (click)="onGroupByChange.emit({key: GroupByKey.RUN})"
+      >
+        <span>
+          <mat-icon
+            *ngIf="selectedGroupBy.key === GroupByKey.RUN"
+            svgIcon="done_24px"
+          ></mat-icon>
+        </span>
+        <label>Runs</label>
```

Comment:
same as `Experiments` 
Want to see if we want to use `runs` instead of `run` from now on